lib/play.js

```diff
@@ -1,11 +1,40 @@
-//Deprecated (under Development, current fixing)
-function smfPlay(data){
-    return new Promise((resolve, reject) => {
-        console.log('Playing...');
-        let midiout = JZZ().openMidiOut(this.config.outputDeviceName);
-        let player = data.smf.player();
-        player.connect(midiout);
-        player.play();
-        resolve(data);
+const JZZ = require('jzz');
+require('jzz-midi-smf')(JZZ);
+const utils = require('./utils');
+
+const smfPlay = (smf, portIdx) => {
+  return new Promise((resolve) => {
```

Comment:
you can remove a bracket and `return` since this function returns Promise directly.